spaceship/lib/spaceship/client.rb

```diff
@@ -498,6 +504,7 @@ def send_shared_login_request(user, password)
           req.headers['X-Apple-Widget-Key'] = self.itc_service_key
           req.headers['Accept'] = 'application/json, text/javascript'
           req.headers["Cookie"] = modified_cookie if modified_cookie
+          req.headers["X-APPLE-HC"] = hashcash if hashcash
```

Comment:
```suggestion
          req.headers["X-Apple-HC"] = hashcash if hashcash
```

I guess this wouldn't matter, but aligning the case pattern to make them consistent feels good. Or did the case matter here?

(I saw `HC` in the header so only `APPLE` -> `Apple`)
<img width="278" alt="Screenshot 2023-02-24 at 11 22 24" src="https://user-images.githubusercontent.com/748949/221167167-e2b999e8-2715-447e-8d0d-48e091edc5ca.png">

Reply:
They got sent up in all caps when I looked via Proxyman but Ill try making them consistent!

Reply:
Haha, I used `curl`. Either or both `curl` and `proxyman` format headers by themselves 😂